src/tictactoe/tictactoe.abi

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
Do we want to include `tictactoe.wasm` and `tictactoe.abi` files along with the source code? @iamveritas @bobgt @halsaphi

Reply:
I say no because user supposed to generate them during this tutorial
having them already might be confusing
unless we want to allow our users to directly deploy to testnet or locally without building... what is our intention here?

Reply:
I'd say remove them too. If the user can't generate the .wasm/.abi files, they have a bigger problem to fix with their toolchain installation or version used.

Reply:
The intention is to guide the user for an end-to-end game contract development. Right from writing the source code > compiling > deploying > pushing actions

I don't think it will hurt providing the wasm and abi as "sample" files just in case the user wants to upload the files in Testnet without doing the compilation for a "quick try. " It would be like providing extra convenience through compiled files.

We can go with either ways.